python/fledge/plugins/north/opcuaclient/opcuaclient.py

```diff
@@ -176,47 +150,39 @@ async def send_payloads(self, payloads):
     async def _send_payloads(self, payload_block):
         """ send a list of block payloads"""
         async with Client(url=config["url"]["value"]) as client:
-
             var = client.get_node(payload_block["node"])
 
-            #_LOGGER.warn("My variable old value %s", await var.read_value())
+            # _LOGGER.warn("My variable old value %s", await var.read_value())
             datavalue = ua.DataValue(self._value_to_variant(payload_block["value"], payload_block["type"]))
```

Comment:
I still had to add _SourceTimestamp_ to the _DataValue_ constructor to get this to work. Were you not intending to address this right now?